recipes-graphics/wayland/weston_%.bbappend

```diff
@@ -0,0 +1,6 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
+SRC_URI += "file://0001-gl-renderer-Add-EGL-client-support-for-EGLStream-fra.patch \
+            file://0002-compositor-Process-stream-attach-requests-with-wl_eg.patch \
+            "
+
```

Comment:
Could you move these patch additions into the `weston_10.0.0.bbappend`?  I expect this got created by devtool, which doesn't handle modifying existing bbappends when they've got version numbers in them.